public_html/wp-content/plugins/wordcamp-speaker-feedback/views/form-feedback.php

```diff
@@ -6,65 +6,105 @@
+		</div>
+
+		<div class="speaker-feedback__field-inline">
+			<label for="sft-author-email">
+				<?php esc_html_e( 'Email', 'wordcamporg' ); ?>
+			</label>
+			<input type="email" id="sft-author-email" name="sft-author-email" required />
+		</div>
+	</div>
+	<?php else : ?>
+		<div class="speaker-feedback__notice">
+			<p><?php
+			$user = wp_get_current_user();
+			echo wp_kses_post( sprintf(
+				__( 'You are leaving feedback as %s.', 'wordcamporg' ),
+				$user->display_name
+			) ); ?></p>
+		</div>
+	<?php endif; ?>
+
+	<div id="speaker-feedback-notice" aria-live="polite" aria-relevant="additions text" aria-atomic="true"></div>
+
+	<div class="speaker-feedback__field">
+		<fieldset class="speaker-feedback__field-rating">
+			<legend><?php esc_html_e( 'Rate this talk', 'wordcamporg' ); ?></legend>
 			<input
 				type="radio"
-				name="feedback-rating"
+				name="sft-rating"
 				value="0"
 				checked
 				aria-label="<?php esc_attr_e( 'No stars', 'wordcamporg' ); ?>"
 			/>
-			<input type="radio" id="sft-rating-val-1" name="feedback-rating" value="1" />
+			<input type="radio" id="sft-rating-val-1" name="sft-rating" value="1" />
 			<label for="sft-rating-val-1">
 				<span class="screen-reader-text"><?php esc_html_e( '1 star', 'wordcamporg' ); ?></span>
 				<?php require get_assets_path() . 'svg/star.svg'; ?>
 			</label>
-			<input type="radio" id="sft-rating-val-2" name="feedback-rating" value="2" />
+			<input type="radio" id="sft-rating-val-2" name="sft-rating" value="2" />
 			<label for="sft-rating-val-2">
 				<span class="screen-reader-text"><?php esc_html_e( '2 stars', 'wordcamporg' ); ?></span>
 				<?php require get_assets_path() . 'svg/star.svg'; ?>
 			</label>
-			<input type="radio" id="sft-rating-val-3" name="feedback-rating" value="3" />
+			<input type="radio" id="sft-rating-val-3" name="sft-rating" value="3" />
 			<label for="sft-rating-val-3">
 				<span class="screen-reader-text"><?php esc_html_e( '3 stars', 'wordcamporg' ); ?></span>
 				<?php require get_assets_path() . 'svg/star.svg'; ?>
 			</label>
-			<input type="radio" id="sft-rating-val-4" name="feedback-rating" value="4" />
+			<input type="radio" id="sft-rating-val-4" name="sft-rating" value="4" />
 			<label for="sft-rating-val-4">
 				<span class="screen-reader-text"><?php esc_html_e( '4 stars', 'wordcamporg' ); ?></span>
 				<?php require get_assets_path() . 'svg/star.svg'; ?>
 			</label>
-			<input type="radio" id="sft-rating-val-5" name="feedback-rating" value="5" />
+			<input type="radio" id="sft-rating-val-5" name="sft-rating" value="5" />
 			<label for="sft-rating-val-5">
 				<span class="screen-reader-text"><?php esc_html_e( '5 stars', 'wordcamporg' ); ?></span>
 				<?php require get_assets_path() . 'svg/star.svg'; ?>
 			</label>
 		</fieldset>
 	</div>
-	
+
 	<div class="speaker-feedback__field">
 		<label for="sft-question-1">
 			<?php esc_html_e( 'What’s one good thing you’d keep in this presentation?', 'wordcamporg' ); ?>
```

Comment:
sure (to this & below) — I didn't want to change the questions around here too much, and I started this PR before that function existed. Is changing that a blocker to merge this PR?

Reply:
No, fine to do this in a separate PR

Reply:
Ended up doing this here after all— 69f3a5cad87a6dfe9df2098a1e6c89250ea6d5e2

Reply:
Oh, since we don't identify field type in `get_feedback_questions`, I can't do a straight loop of these questions — I can refactor that (& the JS) in another PR if you think we need that (I don't think we do, but that's a very weakly-held opinion).